_health-care/_js/containers/NameAndGeneralInfo.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+
+import NameAndGeneralInfoSection from '../components/personal-information/NameAndGeneralInfoSection';
+import { updateField } from '../actions';
+
+function mapStateToProps(state) {
+  console.log(state);
```

Comment:
In fact, with this, we can completely disassociate the shape of the store with the navigation architecture which is what I think @joesoftware was wanting.